.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"commonjs": true,
+		"es6": true,
+		"node": true,
+		"jasmine": true
+	},
+	"plugins": [
+		"react"
+	],
+	"extends": "eslint:recommended",
+	"globals": {
+		"atom": "readonly"
+	},
+	"parserOptions": {
+		"ecmaVersion": 2018,
+		"ecmaFeatures": {
+			"jsx": true,
+		}
+	},
+	"rules": {
+		indent: ["error", "tab"],
```

Comment:
Why tab and not 4 spaces?

Reply:
developer accessibility. With tabs the person looking at the code gets to decide how many characters to indent instead of an arbitrary number set by someone else.

I prefer 2 character indentation but I work on so many different projects I don't even notice when it is different. I just press the <kbd>Tab</kbd> key and let Atom decide if it should enter spaces or a tab character.

Reply:
Yes, Atom decides based on the detected project default.
But i understand it like this is the physical indention.
So i would prefer the defult (4 spaces).

Reply:
should I just leave them all as default?